cherrypy/_cpwsgi.py

```diff
@@ -54,33 +53,36 @@ class VirtualHost(object):
 
         cherrypy.tree.graft(vhost)
     """
-    default = None
-    """Required.
```

Comment:
I think it'd still be nice to have actual class attribute docstrings, not just initializer args.

Reply:
I don't disagree with you, there was just enough changes to make that I didn't put the time into having actual attribute docstrings

Reply:
Yeah.. It might've been easier to merge smaller PRs. I got to review a part of this yesterday, during the sprints, patching it file-by-file. But didn't get through all the files — people around needed help with other stuff they were sprinting on. I was hoping to keep going through it today.